src/api/hooks/useGetWishlist.ts

```diff
@@ -0,0 +1,66 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { BASE_URL, fetchInstance } from '../instance';
+
+export interface WishlistProduct {
+  id: number;
+  name: string;
+  price: number;
+  imageUrl: string;
+}
+
+export interface WishlistItem {
+  id: number;
+  product: WishlistProduct;
+}
+
+export interface WishlistResponseData {
+  content: WishlistItem[];
+  pageable: {
+    sort: {
+      sorted: boolean;
+      unsorted: boolean;
+      empty: boolean;
+    };
+    pageNumber: number;
+    pageSize: number;
+    offset: number;
+    unpaged: boolean;
+    paged: boolean;
+  };
+  totalPages: number;
+  totalElements: number;
+  last: boolean;
+  number: number;
+  size: number;
+  numberOfElements: number;
+  first: boolean;
+  empty: boolean;
+}
+
+export const getWishlistPath = () => `${BASE_URL}/api/wishes`;
```

Comment:
상수가 아닌 함수로 작업하신 이유가 있을까요?
```suggestion
export const WISH_LIST_PATH = `${BASE_URL}/api/wishes`;
```

Reply:
`src/api/hooks/useGetCategorys.ts` 참고해서 코드 작성하다보니 함수로 작성한 것 같아요! 상수로 수정했습니다😊